extensions/blocks/podcast-player/attributes.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * External dependencies
+ */
+import { isUrl } from '@wordpress/url';
+
+export default {
+	url: {
+		type: 'string',
+		validator: isUrl,
```

Comment:
is correct adding this validator here?

Reply:
I copied that from other Jetpack blocks, like [Calendly](https://github.com/Automattic/jetpack/blob/698a72bcac8f38f46e551e8fb7494a23332320aa/extensions/blocks/calendly/attributes.js#L17).

Reply:
Yes, we are using the `validator` key for many Jetpack blocks. However, I don't see any block in the core that uses this key in the block definition.

Reply:
Ah, yes, it's used by Jetpack's [`getValidatedAttributes()`](https://github.com/Automattic/jetpack/blob/ef97833724912a78d17d9f5b0cf094c50a582259/extensions/shared/get-validated-attributes.js)

Reply:
yes, but again the idea of adding the `validator` key to the block definition makes me noise. Not sure if we can modify this structure arbitrarily. Anyway, this is something to keep discussing in another place and context.